tests/1-wire/CMakeLists.txt

```diff
@@ -1,6 +1,17 @@
+add_file_target(FILE wire.v)
+
 add_simple_test(
```

Comment:
Rather than copy paste every test under tests for the QL OpenFPGA stuff, please adjust `add_simple_test` to correctly do the right thing for the QL OpenFPGA arch.  There should be enough data in the arch and device definitions to do so.  The entire reason we use `add_simple_test` instead of `add_fpga_test` is to handle special cases like this.

Reply:
Ok. I've reworked that.